src/components/Product/BestProductSection/BestProductSection.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react';
+import { ProductCard } from '@/components/Product';
+import { baseUrl } from '@/constants/urls';
+import styles from './BestProductSection.module.scss';
+
+const BestProductSection = () => {
+  const [bestProducts, setBestProducts] = useState([]);
+
+  useEffect(() => {
+    const fetchBestProducts = async () => {
+      const res = await fetch(
+        `${baseUrl}/products?page=1&pageSize=1000&orderBy=favorite`, //서버에 좋아요 순으로 정렬한 상위 N개의 데이터를 보내주는 전용 endpoint가 없어서 일단 임시로 이렇게 처리
+      );
```

Comment:
❗️ 수정요청
orderBy param으로 정렬기준을 넘겨주고 pageSize로 원하는 상품 개수를 보내주면 됩니다~ 지금의 경우 아래처럼 작성하시면 될거에요.

```suggestion
      const res = await fetch(`${baseUrl}/products?page=1&pageSize=4&orderBy=favorite`);
```